### PR TITLE
More strict `pre-commit` checks

### PR DESCRIPTION
**Pull request by jmduarte**
- More strict `pre-commit` checks

### CODE REVIEW

**Comment by rkansal47** [comment]
I'll double check if the import order's still an issue.

Are you sure about the 125 line length? I only ask because Black says "people with sight disabilities find it harder to work with line lengths exceeding 100 characters" (although I don't see any study quoted for this). https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length

**Comment by jmduarte** [comment]
Good point. Updated to 100

File: jetnet/__init__.py [hunk 1]
```diff
@@ -2,12 +2,11 @@
 
 # IMPORTANT: evaluation has to be imported first since energyflow must be imported before torch
 # See https://github.com/pkomiske/EnergyFlow/issues/24
+import jetnet.datasets  # noqa: F401
+import jetnet.datasets.normalisations
+import jetnet.datasets.utils
 import jetnet.evaluation
```
[the remark this answers is not in the document]
Comment (rkansal47) [reply jetnet/__init__.py:8]: I'm actually seeing a much weirder issue now - I get a segfault whenever I import pytorch **unless** I import energyflow first. I guess this implies some incompatibility with the version of `libomp` used in torch 1.13? I will look into this further.

This is `macOS 11.4, torch 1.13.0, python 3.10.8, energyflow 1.3.2`.

[the remark this answers is not in the document]
Comment (rkansal47) [reply jetnet/__init__.py:8]: Seems I'd messed up my environment at some point, running in a clean one resolves the `torch` segfault. 

Unfortunately though it seems now that the example I give [here](https://github.com/pkomiske/EnergyFlow/issues/24) doesn't work for me regardless of the import order - I get this `OMP: Error #15: Initializing libomp.dylib, but found libiomp5.dylib already initialized.`


Does this run normally for you:

```
import torch
import energyflow.emd as emd
import numpy as np

x = np.random.rand(100, 30, 4)
y = np.random.rand(100, 30, 4)

emd.emds(x, y)  # gives libomp error
```
?

[the remark this answers is not in the document]
Comment (jmduarte) [reply jetnet/__init__.py:8]: I am able to reproduce the error if I use pure `pip` for dependency management:
```bash
mamba create -n jetnet python=3.10
pip install torch torchvision torchaudio jetnet
```

but, it works for me if i use `conda` for dependency management (except for `jetnet`). that is, i do `mamba env create -f environment.yml` using this `environment.yml`:
```yaml
name: jetnet
channels:
  - conda-forge
  - pytorch
dependencies:
  - python=3.10
  - pytorch
  - torchvision
  - torchaudio
  - pip
  - pip:
    - jetnet
```
so i think it's that `pip` installs `libomp` twice ?

[the remark this answers is not in the document]
Comment (rkansal47) [reply jetnet/__init__.py:8]: Ah indeed uninstalling `pytorch` then re-installing it through `conda` solves this issue. 

Unfortunately though this means if you `pip install jetnet` without installing pytorch through `conda` first, you may run into the `libomp` issue on Macs. 

I believe this is unrelated to this PR so I'll merge this, but I'm not sure what the solution to this issue is - maybe for now we can just add an FAQ for this?

**Review by rkansal47 — APPROVED** [review]
LGTM

**Pull request merged**